tokens/colors.json

```diff
@@ -1,6 +1,7 @@
 [
   { "name": "purple", "hex": "#6e41e0", "shades": [10], "tints": [10] },
   { "name": "blue", "hex": "#1f91cc" },
+  { "name": "radblue", "hex": "#5555FF" },
```

Comment:
Let's use our old palette until we get the go from our design overlords to replace the whole palette in one swoop.